modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -42,4 +44,26 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
       '</td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table><p>x</p>'
     );
   });
+  it('TBA: Fire TableModified event when row are added via Tab key', () => {
+    const editor = hook.editor();
+
+    const events: Array<EditorEvent<TableModifiedEvent>> = [];
+    const logEvent = (event: EditorEvent<TableModifiedEvent>) => {
```

Comment:
Even though this is only used in this test at the moment, I think it would be better to have this setup outside this test and clear the events inside `afterEach`. That way if someone needs to add a test here later, it will be very simple

Reply:
Done